topics/docker.md

```diff
@@ -161,12 +161,33 @@ In the root folder of the project, create a file named `Dockerfile` with the fol
 <tab title="Gradle" group-key="kotlin">
 
 ```Docker
-FROM gradle:7-jdk11 AS build
+# Stage 1: Cache Dependencies
+# 
+# This stage is entirely optional, but helps to 
```

Comment:
Instead of having inline comments here, can you describe this step in the paragraph above?

Reply:
Can  I add this inbetween of `The second stage of the build works in the following way:` and Dockerfile code samples?

```
<tip>
First stage of the both Dockerfile's are optional, but recommended due to
improvement for overall build speed.

If first stage is not used, or dependencies are not cached in other stages,
dependencies declared in ``build.gradle.kts`` will be installed in every build.

If first stage is used, dependencies will be re-downloaded only when there is a change to
``build.gradle.kts``.
</tip>
```